client/src/components/App/Content/InvestigationForm/TabManagement/ClinicalDetails/ClinicalDetails.tsx

```diff
@@ -89,6 +107,12 @@ const ClinicalDetails: React.FC = (): JSX.Element => {
         };
     };
 
+    React.useEffect(() => {
+        setIsInIsolation(new Date(context.clinicalDetailsData.isolationStartDate as Date) !== null || new Date(context.clinicalDetailsData.isolationEndDate as Date) !== null);
```

Comment:
You have a field in the DB that indicates whether the patient is in isolation or not

Reply:
no one told me about any of the new fields, i'll update